src/parser/jobs/mnk/modules/Fists.tsx

```diff
@@ -151,7 +157,7 @@ export class Fists extends Analyser {
 			why: <Trans id="mnk.fists.suggestions.stanceless.why">
 				<Plural value={this.getFistGCDCount(FISTLESS)} one="# GCD" other="# GCDs"	/> had no Fists buff active.
 			</Trans>,
-			tiers: FIST_SEVERITY.DOUSED,
+			tiers: SUGGESTION_TIERS.DOUSED,
```

Comment:
This FISTLESS suggestion can never get hit, since we're unhooking the fistHook whenever a fist is removed, and only re-adding it on applying a fist status (though it looks like that's a pre-existing condition too... has this ever worked?)

Also, it doesn't look like we're doing anything with the GCD count under fists of fire, so we should probably adjust the onCast hook lifetime such that it's active for FISTLESS, Earth, and Wind, but not Fire. It might also make sense to just make the fist GCD counters buckets that always exist and you only have one bucket per relevant fist type, to simplify the getFistGCDCount logic (replace that array filtering & reducing with just accessing the buckets).

Reply:
We don't bother with GCDs under FoF because that's what we want to see usually. FISTLESS looks unused because it'll usually only happen if they forgot to apply a fist at the start. Previously it was mostly a thing in Stormblood post-RoE but we still see it sometimes. Alternatively, could invert the suggestion to count GCDs under FoF and then matchClosestHigher, but we'd need a global GCD count for that which seems excessive to hook.

Bucketing is interesting, but since fists are being removed in 6.0 I'm not sure it's worth the effort. Current MNK gameplay is basically perma-FoF so for the most part we don't see suggestions from this module often anyway.